src/CampaignsAdmin/edit/campaignAdmin.css

```diff
@@ -0,0 +1,21 @@
+.input-continer{
+  display: block;
+  position: relative;
+  cursor: pointer;
+  padding:0px;
+}
+.photo-input{
+  opacity: 0;
+  width: 0.1px;
+  height: 0.1px;
+  position: absolute;
+}
+.photo-input-td{
+  padding: 0 !important;
```

Comment:
This is usually a smell. Why do you need `!important` here?

Reply:
to be applied over the CSS Ant property